src/vcpkg/commands.ci-verify-versions.cpp

```diff
@@ -306,8 +307,10 @@ namespace vcpkg
             if (!manifest_exists && !control_exists)
             {
                 msg::write_unlocalized_text_to_stdout(Color::error, fmt::format("FAIL: {}\n", port_name));
-                errors.emplace(
-                    msg::format(msgPortMissingManifest, msg::package_name = port_name, msg::path = port_path));
+                errors.emplace(LocalizedString::from_raw(port_path)
```

Comment:
I wonder if this pattern will be common enough to warrant a
```cpp
LocalizedString LocalizedString::error_in_path(StringView path) {
    return LocalizedString::from_raw(path).append_raw(": ").append(msgErrorMessage);
}
```

Reply:
I think this would be worth it but I'd rather do it as its own PR rather than expanding the scope of this one